include/nbl/builtin/hlsl/tgmath.hlsl

```diff
@@ -25,19 +26,26 @@ bool isnan(Float val)
 }
 
 template <>
-bool isnan(uint64_t val)
+static inline bool isnan(uint64_t val)
```

Comment:
what does `static inline` do in HLSL or C++ ?

Reply:
in a header file nothing, removed it